index.html

```diff
@@ -1374,6 +1449,20 @@ <h3>
           developer's preferred <a>display mode</a> for the web application.
         </p>
       </section>
+      <section>
+        <h3>
+          <code>display_override</code> member
+        </h3>
+        <p>
+          The <dfn>display_override</dfn> member is a <a>sequence</a> of<a>
+          DisplayModeType</a>. This item represents the developer's preferred
+          fallback chain for display modes. This field overrides the
```

Comment:
Link "fallback chain"

Reply:
I don't see this anywhere, what would you like me to link to?

Reply:
I think you just deleted the concept of a fallback chain :) (making it an implicit part of the algorithm).

In that case, link "display modes" to that section, so there's some way to quickly get from here to the section with the algorithm and details.